parts/k8s/addons/antrea-windows.yaml

```diff
@@ -0,0 +1,189 @@
+apiVersion: v1
+data:
+  Run-AntreaAgent.ps1: |
+    $ErrorActionPreference = "Stop"
+    # wins will rename the binary when executing it. So we need to copy the binary everytime before running it.
+    mkdir -force /host/k/antrea/bin
+    cp /k/antrea/bin/* /host/k/antrea/bin/
+    C:/k/antrea/utils/wins.exe cli process run --path /k/antrea/bin/antrea-agent.exe --args "--config=/k/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=/k/antrea/logs/ --alsologtostderr --log_file_max_size=100 --log_file_max_num=4" --envs "KUBERNETES_SERVICE_HOST=$env:KUBERNETES_SERVICE_HOST KUBERNETES_SERVICE_PORT=$env:KUBERNETES_SERVICE_PORT ANTREA_SERVICE_HOST=$env:ANTREA_SERVICE_HOST ANTREA_SERVICE_PORT=$env:ANTREA_SERVICE_PORT NODE_NAME=$env:NODE_NAME"
```

Comment:
Can we get this configured, installed and running without wins?  We don't use wins to configure the rest of the components and currently don't want to introduce the component.

Reply:
Hi @jsturtevant , Antrea depends on wins to manage antrea-agent and kube-process processes from Pod.
The way is also the official recommendation: https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/

Are you just want avoid to introduce wins or any this kind of tools? Without wins like tools we will must start antrea-agent process directly from host and do lots of configuration for it.

Reply:
We don't want to introduce wins at this point for aks-engine.  Wins is a different way of configuring a cluster and aks-engine comes from a time before that was a way of doing things.  Introducing a second way of configuring things is out of scope for aks-engine at this point.

Reply:
OK, get it. We will add scripts help to start antre-agent without wins.

Reply:
thanks

Reply:
There currently isn't a way to start a process on the host from a POD.  Services running on the host still have access to the k8s control plane, this is the way kubelet and the other cni's are running currently.  

There is a draft KEP related to enabling privileged containers to solve this problem outside of k8s. Please add some comments to make sure it fits your needs: https://docs.google.com/document/d/12EUtMdWFxhTCfFrqhlBGWV70MkZZPOgxw0X-LTR0VAo/edit#heading=h.kpsimyb2q1sg

Reply:
@jsturtevant We have added antrea as a windows service. Could you please take a look at the updated review. Hopefully we can checkin soon.

I ran E2E tests on Windows and it passed with Antrea v0.9.0